### PR TITLE
Project import generated by Copybara.

### PR DESCRIPTION
**Pull request by cstirdivant**
Project import generated by Copybara.

### CODE REVIEW

**Comment by cstirdivant** [comment]
@charbull please review :)

**Comment by cstirdivant** [comment]
@charbull friendly ping :)

File: ontology/docs/building_config.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Model Instance Configuration (building configuration file)
```
Comment (charbull) [inline ontology/docs/building_config.md:1]: this file should not be synched currently as the code is not synched. revert this part?

**Review by charbull — CHANGES_REQUESTED** [review]

File: ontology/docs/building_config.md (hunk 1 above)
Comment (cstirdivant) [reply ontology/docs/building_config.md]: ty - reverted

**Comment by cstirdivant** [comment]
@charbull changes made - please review

**Review by charbull — APPROVED** [review]

**Pull request merged**